sql/branch.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
+    -- Tips that have been set to zero should have zero dues.
+
+    UPDATE payment_instructions SET due = 0 WHERE amount = 0 AND due != 0;
+
+    -- If a user has changed the tip amount at any point of time - we don't have a
+    -- way to properly repair the due, so we set it to zero.
+
+    UPDATE payment_instructions pi
+       SET due = 0
+     WHERE due > 9.41
+       AND (
+             SELECT COUNT(*)                    -- Tips with a different amount exist.
+               FROM payment_instructions pi2
+              WHERE pi.participant = pi2.participant
+                AND pi.team = pi2.team
+                AND amount != pi.amount
+           ) > 0;
+
+    -- For the rest of the payment_instructions, we set due to the highest multiple of the
+    -- tip amount that is lower than the minimum charge.
+
+    UPDATE payment_instructions SET due = floor(9.41/(amount)) * amount WHERE due > 9.41;
```

Comment:
> Shouldn't it be the lowest multiple that is higher?

Ah, yes. True. We charge their CC the day they exceed the minimum charge. 

> What about people who had two tips set up? We compute minimum due based on all payment instructions together 

Hmm, this is starting to get tricky..